--- orig.py	2022-06-23 12:31:28.000000000 +0100
+++ new.py	2022-06-23 12:31:45.000000000 +0100
@@ -2189,7 +2189,7 @@
         typ = "".join(n.astext() for n in self.render(node.get_type()))
         # in Doxygen < 1.9 the 'friend' part is there, but afterwards not
         # https://github.com/michaeljones/breathe/issues/616
-        assert typ in ("friend class", "friend struct", "class", "struct")
+        #assert typ in ("friend class", "friend struct", "class", "struct")
         if not typ.startswith("friend "):
             typ = "friend " + typ
         signode += addnodes.desc_annotation(typ, typ)
